riscv/extension.h

```diff
@@ -11,21 +11,19 @@
 class extension_t
 {
  public:
-  virtual std::vector<insn_desc_t> get_instructions() = 0;
-  virtual std::vector<disasm_insn_t*> get_disasms() = 0;
-  virtual std::vector<csr_t_p> get_csrs ([[maybe_unused]] processor_t &proc) const { return {}; };
-  virtual const char* name() = 0;
-  virtual void reset() {};
-  virtual void set_debug(bool UNUSED value) {}
-  virtual ~extension_t();
+   virtual std::vector<insn_desc_t> get_instructions(processor_t &proc) = 0;
+   virtual std::vector<disasm_insn_t *>
+   get_disasms(processor_t *proc = nullptr) = 0;
```

Comment:
Can you split the formatting into a different commit? Or omit it altogether from this PR (This didn't seem to do the right thing here I think)